Gemfile

```diff
@@ -23,6 +23,10 @@ end
 
 gem 'simplecov', '~> 0.8'
 
+if RUBY_VERSION < '2.0.0' || RUBY_ENGINE == 'java'
+  gem 'json', '< 2.0.0'
+end
```

Comment:
Why do we need to specify the `json` gem in the `Gemfile` for each gem?  Does rspec-mocks use it?